### PR TITLE
jwt  generated after login at backend

### PR DESCRIPTION
**Pull request by balani98**

### CODE REVIEW

**Comment by sonalichawla16** [comment]
resolve conflicts

**Comment by balani98** [comment]
i have resolved all the conflicts

File: api/controllers/login.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            },
+            config.get("jwtPrivateKey"),
+            { expiresIn: 86400 }
+          );
+         if(employeeObj!=null){
+            res.status(200).send({
+                success:true,
+                   payload:{
+                     message:"this employee exists in db",
+                     data:{
```
Comment (sonalichawla16) [inline api/controllers/login.js:30]: Please correct casing

File: api/controllers/login.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                        "x-auth-token":token
+                     }
+                  }
+           })
+        }else{
+           res.status(401).send({
+              success:true,
+                 payload:{
+                   message:"you are unauthorized on ARMS"
+                }
```
Comment (sonalichawla16) [inline api/controllers/login.js:40]: Please correct casing

File: web/src/app/jd-form/jd-form.component.ts [hunk 3]
```diff
@@ -26,21 +45,44 @@ export class JdFormComponent implements OnInit {
...
+       jobProfileDescription: this.jobProfileDescription.nativeElement.value,
+       skills: this.skills.nativeElement.value,
+       jobType: this.jobType.nativeElement.value,
+       eligibilityCriteria: this.eligibilityCriteria.nativeElement.value,
+       location: this.location.nativeElement.value,
+       salary: this.salary.nativeElement.value,
+        vacancies: this.vacancies.nativeElement.value,
+     }
+     console.log(this.jdFormObject);
+     this.router.navigate(["/jd-pdf"])
```
Comment (sonalichawla16) [inline web/src/app/jd-form/jd-form.component.ts:78]: Remove console logs please

File: api/controllers/login.js (hunk 1 above)
Comment (balani98) [reply api/controllers/login.js:30]: changed to Pascal casing

File: api/controllers/login.js (hunk 2 above)
Comment (balani98) [reply api/controllers/login.js:40]: same ..

File: web/src/app/jd-form/jd-form.component.ts (hunk 3 above)
Comment (balani98) [reply web/src/app/jd-form/jd-form.component.ts:78]: removed

**Pull request merged**